File: backend/src/Shalom.Application/Contracts/PrayerFocusDto.cs
namespace Shalom.Application.Contracts;

/// <summary>
/// Today's prayer focus for the morning ritual: a deterministic rotation
/// over the active people plus two fixed focuses ("your church",
/// "FaithTech") keyed by day-of-year, with a warm one-sentence framing per
/// relationship bucket. <paramref name="TomorrowName"/> powers the evening
/// "Tomorrow" glance without a second round trip.
/// </summary>
public record PrayerFocusDto(string Name, string Line, string TomorrowName);

File: frontend/projects/api/src/lib/models/prayer-focus-dto.ts
/**
 * Today's prayer focus for the morning ritual — a deterministic server-side
 * rotation over the active people plus two fixed focuses ("your church",
 * "FaithTech") keyed by day-of-year. `line` is the warm one-sentence framing
 * for the focus; `tomorrowName` powers the evening "Tomorrow" glance without
 * a second round trip. Names always come from data, never hardcoded.
 */
export interface PrayerFocusDto {
  readonly name: string;
  readonly line: string;
  readonly tomorrowName: string;
}
